vendor/bat-native-ledger/include/bat/ledger/publisher_info.h

```diff
@@ -15,7 +15,7 @@
 
 namespace ledger {
 
-const std::string clear_favicon = "clear";
+const char clear_favicon[] = "clear";
```

Comment:
nit: globals should start with _, i.e. `_clear_favicon`

Reply:
done